client/src/Movies/MovieList.js

```diff
@@ -20,35 +21,19 @@ export default class MovieList extends Component {
       });
   }
 
+
+
   render() {
     return (
       <div className="movie-list">
         {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
+          <div key={movie.id}>
+            <MovieCard movie={movie} />
```

Comment:
replace with
```suggestion
            <MovieCard key={movie.id} movie={movie} addToSavedList={this.props.addToSavedList} />
```
I believe this is your bug.